kahuna/public/js/components/gr-add-label/gr-add-label.html

```diff
@@ -1,4 +1,5 @@
-<button ng-class="{'small': ctrl.grSmall}"
+<button id="it-add-label-button"
```

Comment:
I think this component is used on each cell in the search page. That is, this will produce invalid HTML as IDs should be unique in a DOM tree.

![image](https://user-images.githubusercontent.com/836140/80109692-ae4a6380-8575-11ea-9ca5-c9ed97d7d496.png)